NEWS.md

```diff
@@ -5,3 +5,47 @@
 * Prevent `init()` in home directory.
 * Included `build_js()` and `build_sass()` in template CI.
 * Use R version from the lockfile in CI.
+
+# rhino 1.1.1
```

Comment:
Let's link the section titles to GitHub releases (ditto for other titles).
```suggestion
# [rhino 1.1.1](https://github.com/Appsilon/rhino/releases/tag/v1.1.1)
```

Reply:
Added Github release links to sections titles.